docs/stdlib/datetime.rst

```diff
@@ -461,29 +460,27 @@ EdgeDB stores and outputs timezone-aware values in UTC.
       ...        <cal::local_datetime>'2021-01-30T23:00:00' + m;
       {false}
 
-    See functions :eql:func:`cal::to_relative_duration`, and :eql:func:`to_str`
-    and date/time :eql:op:`operators <dtminus>` for more ways of working with
-    :eql:type:`cal::relative_duration`.
+    For more information regarding interacting with this type, see
+    :eql:func:`cal::to_relative_duration`, :eql:func:`to_str` and
+    :eql:op:`operators <dtminus>`.
 
 
 ----------
 
 
 .. eql:type:: cal::date_duration
 
-    A type representing a span of time in days.
+    Represents a timespan of a date.
 
     .. note::
 
       This type is only available in EdgeDB 2.0 or later.
 
-
-    The ``date_duration`` type is similar to ``relative_duration``, but it
-    only uses 2 different units under the hood: months and days. It is the
-    result of subtracting one :eql:type:`cal::local_date` from another. The
-    purpose of this type is to allow performing ``+`` and ``-`` operations on
-    :eql:type:`cal::local_date` and produce :eql:type:`cal::local_date` as the
-    result.
+    This type is similar to :eql:type:`cal::relative_duration`, except it only
+    uses 2 units: months and days. This is the result of subtracting one
```

Comment:
```suggestion
    uses 2 units: months and days. It is the result of subtracting one
```

"This" sounds like it might refer to the type's use of only the two units instead of to the type itself. "It" here would more obviously refer to the type itself.